src/LinearMaps.jl

```diff
@@ -3,14 +3,13 @@ module LinearMaps
 
 export LinearMap, AbstractLinearMap
 
-import Base: +, -, *, \, /, ==, transpose
-
-if VERSION >= v"0.7.0-DEV.1415"
-    const adjoint = Base.adjoint
-else
-    const adjoint = Base.ctranspose
-end
+import LinearAlgebra, SparseArrays
 
+import Base: +, -, *, \, /, ==
+import LinearAlgebra: transpose, adjoint
+import LinearAlgebra: issymmetric, ishermitian, isposdef
+import LinearAlgebra: BLAS.axpy!, mul!, lmul!, UniformScaling
+import SparseArrays: sparse
```

Comment:
I don't think we provide any new method definition for `UniformScaling`, so do we need that one?